### PR TITLE
[DOCS] Write TextureButton, review and fill missing entries in TextureProgress

### PR DESCRIPTION
**Pull request by NathanLovato**

### CODE REVIEW

File: doc/classes/TextureProgress.xml [hunk 1]
```diff
@@ -151,51 +151,59 @@
...
+			See [member Range.value], [member Range.max_value].
 		</member>
 		<member name="radial_initial_angle" type="float" setter="set_radial_initial_angle" getter="get_radial_initial_angle">
-			Start angle for radial mode.
+			Starting angle for the fill of [member texture_progress] if [member fill_mode] is [code]FILL_CLOCKWISE[/code] or [code]FILL_COUNTER_CLOCKWISE[/code]. When the node's [code]value[/code] is equal to its [code]min_value[/code], the texture doesn't show up at all. When the [code]value[/code] increases, the texture fills and tends towards [member radial_fill_degrees].
 		</member>
 		<member name="stretch_margin_bottom" type="int" setter="set_stretch_margin" getter="get_stretch_margin">
-			Nine-patch texture offset for bottom margin.
+			The height of the 9-patch's bottom row. A margin of 16 means the 9-slice's bottom corners and side will have a height of 16 pixels. You can set all 4 margin values indivually to create panels with non-uniform borders.
```
Comment (Noshyaar) [inline doc/classes/TextureProgress.xml]: individually

File: doc/classes/TextureButton.xml [hunk 2]
```diff
@@ -127,36 +127,51 @@
...
+			The texture keeps its original size and stays in the bounding rectangle's top-left corner.
 		</constant>
 		<constant name="STRETCH_KEEP_CENTERED" value="3">
+			The texture keeps its original size and stays centered in the node's bounding rectangle.
 		</constant>
 		<constant name="STRETCH_KEEP_ASPECT" value="4">
+			Scale the texture to fit the node's bounding rectangle, but maintain the texture's aspect ratio.
 		</constant>
 		<constant name="STRETCH_KEEP_ASPECT_CENTERED" value="5">
+			Scale the texture to fit the node's bounding rectangle, center it and maintain its aspect ratio.
```
Comment (cbscribe) [inline doc/classes/TextureButton.xml]: Oxford comma here: ", and maintain"

File: doc/classes/TextureButton.xml [hunk 3]
```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="TextureButton" inherits="BaseButton" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
-		Button that can be themed with textures.
+		Texture based button. Supports Pressed, Hover, Disabled and Focused states.
```
Comment (cbscribe) [inline doc/classes/TextureButton.xml]: Should be "Texture-based"

File: doc/classes/TextureProgress.xml [hunk 4]
```diff
@@ -151,51 +151,59 @@
 			The fill direction. Uses FILL_* constants.
 		</member>
 		<member name="nine_patch_stretch" type="bool" setter="set_nine_patch_stretch" getter="get_nine_patch_stretch">
-			If [code]true[/code] textures will be stretched as [NinePatchRect]. Uses [code]stretch_margin[/code] properties (see below). Default value: [code]false[/code]
+			If [code]true[/code], Godot treats the bar's textures like [NinePatchRect]. Use [code]stretch_margin_*[/code], like [member stretch_margin_bottom], to set up the nine patch's 3x3 grid. Default value: [code]false[/code].
```
Comment (cbscribe) [inline doc/classes/TextureProgress.xml]: Remove the comma

**Review by cbscribe — CHANGES_REQUESTED** [review]

**Comment by NathanLovato** [comment]
All fixed!

File: doc/classes/TextureButton.xml [hunk 5]
```diff
@@ -127,36 +127,51 @@
 	</methods>
 	<members>
 		<member name="expand" type="bool" setter="set_expand" getter="get_expand">
+			If [code]true[/code] the texture stretches to the edges of the node's bounding rectangle using the [member stretch_mode]. If [code]false[/code] the texture will not scale with the node. Default value: [code]false[/code].
 		</member>
 		<member name="stretch_mode" type="int" setter="set_stretch_mode" getter="get_stretch_mode" enum="TextureButton.StretchMode">
+			Controls the texture's behavior when you resize the node's bounding rectangle, [b]only if[/b] [member expand] is [code]true[/code]. Set it to one of the [code]STRETCH_*[/code] constants. See the constants to learn more.
 		</member>
 		<member name="texture_click_mask" type="BitMap" setter="set_click_mask" getter="get_click_mask">
+			Pure black and white [Bitmap] image to use for click detection. On the mask, white pixels represent the button's clickable area. Use it to create buttons with curved shapes.
```
Comment (djrm) [inline doc/classes/TextureButton.xml]: isnt this [BitMap]

**Review by cbscribe — APPROVED** [review]

**Pull request merged**